src/test/java/me/karakelley/tictactoe/UITest.java

```diff
@@ -0,0 +1,20 @@
+package me.karakelley.tictactoe;
+
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class UITest {
+
+    @Test
+    public void welcomeMessage() {
+        final ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+        System.setOut(new PrintStream(outContent));
```

Comment:
Will this cause test pollution between other tests? I think there's a better way to test this than to override the global stdout `PrintStream`.